### PR TITLE
add scrollview bar parse for cocostudio in reader 

### PR DESCRIPTION
**Pull request by geron-cn**
add scrollview bar parse for cocostudio in reader 
 update ccs-res for scroll bar test  
*\* need merge https://github.com/dumganhar/ccs-res/pull/56 **

### CODE REVIEW

**Comment by geron-cn** [comment]
@dumganhar @xiaofeng11   please review this pr, thank you.

File: cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp [hunk 1]
```diff
@@ -118,7 +118,9 @@ namespace cocostudio
         Size innerSize(200, 300);
         int direction = 0;
         bool bounceEnabled = false;
-        
+		bool scrollbarEnabled = true;
+		bool scrollbarAutoHide = true;
+		float scrollbarAutoHideTime = 0.2f;
```
Comment (xiaofeng11) [inline cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp]: Please fix the indent

File: cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp [hunk 2]
```diff
@@ -129,7 +131,7 @@ namespace cocostudio
             
             if (name == "ClipAble")
             {
-                clipEnabled = (value == "True") ? true : false;
+				clipEnabled = FLATSTR_TO_BOOL(value);
```
Comment (xiaofeng11) [inline cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp]: Please fix the indent

File: cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp [hunk 3]
```diff
@@ -141,10 +143,7 @@ namespace cocostudio
             }
             else if (name == "Scale9Enable")
             {
-                if (value == "True")
-                {
-                    backGroundScale9Enabled = true;
-                }
+				backGroundScale9Enabled = FLATSTR_TO_BOOL(value);
```
Comment (xiaofeng11) [inline cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp]: Please fix the indent

File: cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp [hunk 4]
```diff
@@ -179,9 +178,20 @@ namespace cocostudio
             }
             else if (name == "IsBounceEnabled")
             {
-                bounceEnabled = (value == "True") ? true : false;
+				bounceEnabled = FLATSTR_TO_BOOL(value);
```
Comment (xiaofeng11) [inline cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp]: Please fix the indent

File: cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp [hunk 5]
```diff
@@ -179,9 +178,20 @@ namespace cocostudio
...
+				scrollbarEnabled = FLATSTR_TO_BOOL(value);
+			}
+			else if (name.compare("IsScrollBarAutoHide") == 0)
+			{
+				scrollbarAutoHide = FLATSTR_TO_BOOL(value);
+			}
+			else if (name.compare("ScrollbarAutoHideTime") == 0)
+			{
+				scrollbarAutoHideTime = atof(value.c_str());
+			}
```
Comment (xiaofeng11) [inline cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp]: Please fix the indent

File: cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp [hunk 6]
```diff
@@ -392,7 +402,10 @@ namespace cocostudio
                                                backGroundScale9Enabled,
                                                &f_innerSize,
                                                direction,
-                                               bounceEnabled);
+                                               bounceEnabled,
+											   scrollbarEnabled,
+											   scrollbarAutoHide,
+											   scrollbarAutoHideTime);
```
Comment (xiaofeng11) [inline cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp]: Please fix the indent

File: cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp [hunk 7]
```diff
@@ -512,7 +525,16 @@ namespace cocostudio
...
+
+		bool scrollbarEnabled = options->scrollbarEnabeld() != 0;
+		scrollView->setScrollBarEnabled(scrollbarEnabled);
+		bool scrollbarAutoHide = options->scrollbarAutoHide() != 0;
+		if (scrollbarEnabled)
+		{
+			scrollView->setScrollBarAutoHideEnabled(scrollbarAutoHide);
+			float barAutoHideTime = options->scrollbarAutoHideTime();
+			scrollView->setScrollBarAutoHideTime(barAutoHideTime);
+		}
```
Comment (xiaofeng11) [inline cocos/editor-support/cocostudio/WidgetReader/ScrollViewReader/ScrollViewReader.cpp]: Please fix the indent

**Comment by geron-cn** [comment]
[ci skip]

**Pull request merged**